frontend/src/libs/components/infinite-scroll/infinite-scroll.tsx

```diff
@@ -0,0 +1,31 @@
+import ReactInfiniteScroll from 'react-infinite-scroll-component';
+
+type Properties = {
+  children: React.ReactNode;
+  hasMore: boolean;
+  dataLength: number;
+  fetchData: () => void;
+  className?: string;
+};
+
+const InfiniteScroll: React.FC<Properties> = ({
+  fetchData,
+  children,
+  hasMore,
+  dataLength,
+  className,
+}) => {
```

Comment:
```suggestion
}) => (
    <ReactInfiniteScroll
      loader={null}
      next={fetchData}
      hasMore={hasMore}
      dataLength={dataLength}
      className={className}
    >
      {children}
    </ReactInfiniteScroll>
  );
```

Reply:
I'll update it here https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/187